<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sell</class>
 <widget class="QMainWindow" name="Sell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sell</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>1061</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="managePage">
     <widget class="QLabel" name="WaitForSend">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>待发货：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSendName">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>商品名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sendName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSendNummber">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>商品数量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="sendLeft">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>130</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>库存剩余0000</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSendPrice">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>170</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>提交价格：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sendPrice">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSendSum">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>商品总价：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sendSum">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSendAddres">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>买家地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sendAddress">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>250</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSendSelectSend">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>物流：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sendSelect">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>顺丰</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>京东</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>申通</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>圆通</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="send">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>发货</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSendData">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>商品货号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="waitForPay">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>待付款：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sendData">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPayData">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>商品货号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPayName">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>80</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>商品名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="payName">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>80</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="payData">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>40</y>
        <width>87</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPayNumber">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>商品数量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPayLeft">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>库存剩余0000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="remindPay">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>310</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>提醒对方付款</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sendNumber">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="payNumber">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>180</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>买家地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPayBuyerAddress">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>180</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="readInformation">
     <widget class="QPushButton" name="btnCorrectCommodityINform">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>400</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>修改商品信息</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSellNumber">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>销量：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>折扣：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>优惠券：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditGreaterThanPrice">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>330</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditMinusPrice">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>330</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>满</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>330</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>减</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>250</width>
        <height>250</height>
       </rect>
      </property>
      <property name="text">
       <string>图像</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>名称：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCommodityName">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>120</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSellData">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>150</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>0000</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDiscount">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>260</y>
        <width>561</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>统一折扣</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPrice_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>180</y>
        <width>157</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelPrice_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>价格：</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLeftNumber_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>45</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>数量：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLeftNumber">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>不统一折扣</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageShowCommodityList">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1001</width>
        <height>581</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>999</width>
         <height>579</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="sellRecord">
     <widget class="QListWidget" name="dailyList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1001</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="correctInformatiion">
     <widget class="QPushButton" name="sureChangeUserInfBtn">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QLabel" name="passWordLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>新密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="userNameLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
     <widget class="QLabel" name="genderLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>地址：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelChangeUserInfBtn">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userNamelineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordlineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sureLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="surelineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="primPassWordlineEdit">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="primPasswordlabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>原密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAddress">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1021</width>
        <height>541</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1019</width>
         <height>539</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1011</width>
          <height>531</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>550</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>一键清空</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonReadInform">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>修改商品信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCorrectInform">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>修改用户信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSellRecord">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>销售统计</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSellManage">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>商品管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonQiut">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>退出程序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>通知</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionReadInform">
   <property name="text">
    <string>查看信息</string>
   </property>
  </action>
  <action name="actionCorrectInform">
   <property name="text">
    <string>修改信息</string>
   </property>
  </action>
  <action name="actionSellRecord">
   <property name="text">
    <string>销售统计</string>
   </property>
  </action>
  <action name="actionManage">
   <property name="text">
    <string>商品管理</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
